packages/cli/src/oclif/commands/login.js

```diff
@@ -19,7 +19,13 @@ const { writeFile } = require('../../utils/files');
 const { prettyJSONstringify } = require('../../utils/display');
 const { isSamlEmail } = require('../../utils/credentials');
 
-const DEPLOY_KEY_DASH_URL = `${BASE_ENDPOINT}/developer/partner-settings/deploy-keys/`;
+const getDeployKeyUrl = () => {
+  const url = new URL(BASE_ENDPOINT);
+  url.hostname = `developer.${url.hostname}`;
+  url.pathname = 'partner-settings/deploy-keys/';
+  return url.href;
+};
+const DEPLOY_KEY_DASH_URL = getDeployKeyUrl();
```

Comment:
Non-blocking: any reason not to assign `DEPLOY_KEY_DASH_URL` directly to the output of the function? e.g.

```javascript
const DEPLOY_KEY_DASH_URL = () => {
  const url = new URL(BASE_ENDPOINT);
  url.hostname = `developer.${url.hostname}`;
  url.pathname = 'partner-settings/deploy-keys/';
  return url.href;
};
```

Reply:
less changes, bc then i'd have to invoke it where it's used 😅